ui/packages/consul-ui/package.json

```diff
@@ -176,6 +176,7 @@
   "resolutions": {
     "ember-basic-dropdown": "^3.0.10",
     "ember-changeset": "3.10.1",
+    "refractor": "^3.5.0",
```

Comment:
Why did you need to specify a resolution for refractor? Yarn not figuring out a common version?

Reply:
Oh let me double check that. I had trouble getting a version of `refractor` that had the mermaid lang that came with it for free (they added it in 3.5.0). I _think_ I had to do this to get docfy to internally use the same version and hence pick up the syntax alias additions.

I also couldn't just upgrade docfy (which I think also has an upgraded refractor) as I got a bunch of embroider errors 😬 

I might be completely misremembering tho (and I only did it this morning!) so I'll triple check. Thanks for the spot!

Reply:
Ah right, so in the end it was the `@mapbox/rehype-prism` upgrade that made things work in the end (I did misremember 😆 ). The resolution was a previous attempt. Thanks for the nudge! I just removed it.

(the problem here was, once I'd upgraded refractor, when running docfy it wasn't picking up my refactor syntax aliases. I tried upgrading docfy but I ended up in embroider errors so backed out of that. Messed around with resolutions incase it was somehow using two instances of refractor or something, but then when I figured rehype-prism was in the mix I figured an upgrade might make things magically work there - and it did - but then forgot to remove the resolution)